src/app/products/loan-products/loan-product-stepper/loan-product-interest-refund-step/string-enum-option-data-model.ts

```diff
@@ -0,0 +1,5 @@
+export interface StringEnumOptionData {
```

Comment:
Already exists an interface for that: OptionData in 'app/shared/models/option-data.model'

Reply:
Its not the same. OptionData id is number. StringEnumOptionData id is string